akka-stream-tests/src/test/scala/akka/stream/io/FileSourceSpec.scala

```diff
@@ -59,6 +59,20 @@ class FileSourceSpec extends StreamSpec(UnboundedMailboxConfig) {
     f
   }
 
+  val directoryInsteadOfFile = {
+    val f = Files.createTempDirectory(fs.getPath("/"), "directory-instead-of-file")
```

Comment:
Unnecessary inner val? You're just returning it immediately anyway.

Reply:
Sorry, that's daft - will just return.